tests/suites/0_stateless/20+_others/20_0001_planner_v2.sql

```diff
@@ -240,4 +240,9 @@ select '====Correlated Subquery====';
 select * from numbers(10) as t where exists (select * from numbers(2) as t1 where t.number = t1.number);
 select (select number from numbers(10) as t1 where t.number = t1.number) from numbers(10) as t order by number;
 
+-- position function
+select '===Position Function===';
+SELECT POSITION('bar' IN 'foobarbar');
```

Comment:
Shall we allow it to take variable as argument? Like `position('bar' in column)`

Reply:
I checked the old planner, it supports the case.

Reply:
This works in pg:

```SQL
drop table if EXISTS T;
create table t (a varchar, B varchar);
insert into t values ('FOO', 'barfoo');
select position(UPPER(A) in UPPER(B)) from T;
```